From a185e958b5614ea7e606e85d3de879c5ffc127c0 Mon Sep 17 00:00:00 2001
From: Stefano Lattarini <stefano.lattarini@gmail.com>
Date: Thu, 30 Aug 2012 18:53:13 +0200
Subject: [PATCH 10/89] build: prefer $(AM_CPPFLAGS) over $(INCLUDES)

The latter is obsolete, and might be removed in future Automake versions.
Moreover, it's already been removed in Automake-NG, so its use would make
a port to Automake-NG more difficult.

* parted/Makefile.am (INCLUDES): Rename ...
(AM_CPPFLAGS): ... like this.
* partprobe/Makefile.am: Likewise.
* libparted/fs/Makefile.am: Likewise.
* libparted/Makefile.am: Likewise.
* libparted/labels/Makefile.am: Likewise.

Copyright-paperwork-exempt: yes
---
 libparted/Makefile.am        | 2 +-
 libparted/fs/Makefile.am     | 2 +-
 libparted/labels/Makefile.am | 2 +-
 parted/Makefile.am           | 2 +-
 partprobe/Makefile.am        | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/libparted/Makefile.am b/libparted/Makefile.am
index 7cf35b2..bdc7976 100644
--- a/libparted/Makefile.am
+++ b/libparted/Makefile.am
@@ -62,4 +62,4 @@ libparted_la_LIBADD =	\
 
 EXTRA_DIST	      = mbr.s
 
-INCLUDES = $(partedincludedir) $(INTLINCS)
+AM_CPPFLAGS = $(partedincludedir) $(INTLINCS)
diff --git a/libparted/fs/Makefile.am b/libparted/fs/Makefile.am
index 8d48ea1..7fe364a 100644
--- a/libparted/fs/Makefile.am
+++ b/libparted/fs/Makefile.am
@@ -114,7 +114,7 @@ libparted_fs_resize_la_SOURCES = \
   r/hfs/reloc_plus.c		\
   r/hfs/reloc_plus.h
 
-INCLUDES = \
+AM_CPPFLAGS = \
   -I$(top_srcdir)/libparted/labels	\
   $(partedincludedir)			\
   $(INTLINCS)
diff --git a/libparted/labels/Makefile.am b/libparted/labels/Makefile.am
index 7fe347b..ec8abeb 100644
--- a/libparted/labels/Makefile.am
+++ b/libparted/labels/Makefile.am
@@ -37,7 +37,7 @@ liblabels_la_SOURCES = \
 
 liblabels_la_LIBADD = $(OS_LIBS) $(INTLLIBS)
 
-INCLUDES = $(partedincludedir) $(INTLINCS)
+AM_CPPFLAGS = $(partedincludedir) $(INTLINCS)
 
 BUILT_SOURCES = pt-limit.c
 MAINTAINERCLEANFILES = $(BUILT_SOURCES)
diff --git a/parted/Makefile.am b/parted/Makefile.am
index 0b23693..e7bba2e 100644
--- a/parted/Makefile.am
+++ b/parted/Makefile.am
@@ -48,6 +48,6 @@ parted_LDFLAGS = $(PARTEDLDFLAGS)
 # Tell the linker to omit references to unused shared libraries.
 parted_LDFLAGS += $(IGNORE_UNUSED_LIBRARIES_CFLAGS)
 
-INCLUDES	= $(partedincludedir) $(INTLINCS)
+AM_CPPFLAGS = $(partedincludedir) $(INTLINCS)
 
 MAINTAINERCLEANFILES += Makefile.in
diff --git a/partprobe/Makefile.am b/partprobe/Makefile.am
index dbe2c9f..c0304a0 100644
--- a/partprobe/Makefile.am
+++ b/partprobe/Makefile.am
@@ -13,4 +13,4 @@ partprobe_LDADD = \
 
 partprobe_LDFLAGS = $(PARTEDLDFLAGS)
 
-INCLUDES	= $(partedincludedir) $(INTLINCS)
+AM_CPPFLAGS = $(partedincludedir) $(INTLINCS)
-- 
1.8.5.3

